Mage.Sets/src/mage/cards/m/MaskwoodNexus.java

```diff
@@ -81,31 +82,31 @@ public boolean apply(Game game, Ability source) {
         }
         // Creature cards you own that aren't on the battlefield
         // in graveyard
-        for (UUID cardId : controller.getGraveyard()) {
-            Card card = game.getCard(cardId);
-            if (card != null && card.isCreature(game)) {
-                game.getState().getCreateMageObjectAttribute(card, game).getSubtype().setIsAllCreatureTypes(true);
-            }
-        }
+        Set<Card> affectedCards =
+                controller.getGraveyard().stream()
+                        .map(game::getCard).filter(Objects::nonNull)
+                        .filter(card -> card.isOwnedBy(controller.getId()) && card.isCreature(game))
```

Comment:
no need for owner check on zones which will only contain owned cards (graveyard, hand, library)

and as mentioned elsewhere, is it really correct to check if the card is a creature, since you're checking later whether the object part is a creature?